Dockerfile-full

```diff
@@ -12,7 +12,7 @@ WORKDIR /usr/src/app
 COPY ./src/main/resources/public/package.json /usr/src/app/
 RUN npm install
 COPY ./src/main/resources/public/ /usr/src/app/
-RUN npm run test && npm run build
+RUN npm run build && npm run test
```

Comment:
Doesn't look like a relevant change.

Reply:
Please see the comment https://github.com/reportportal/service-ui/pull/1982#issuecomment-530579950